dss/util/json_gen/generator.py

```diff
@@ -182,19 +182,25 @@ def _object(self, schema: dict) -> dict:
         return impostor
 
     def _number(self, schema: dict) -> Union[int, float]:
+        '''
+        Note: schemas with a multipleOf float value are not supported due to rounding errors in floating point
+        operations.
+        '''
         impostor = self._common(schema)
         if impostor is None:
             maximum = schema.get('maximum', schema.get('exclusiveMaximum', self.UNBOUND_MAX_INT) - 1)
             minimum = schema.get('minimum', schema.get('exclusiveMinimum', self.UNBOUND_MIN_INT) + 1)
             if minimum == maximum:
                 impostor = minimum
             else:
-                multiple_of = schema.get('multipleOf', 1)
-                if isinstance(maximum, float) or isinstance(minimum, float) or isinstance(minimum, float):
+                multiple_of = schema.get('multipleOf')
+                if isinstance(maximum, float) or isinstance(minimum, float):
                     impostor = self.faker.random.uniform(minimum, maximum)
                 else:
                     impostor = self.faker.random_int(minimum, maximum)
-                impostor = impostor // multiple_of * multiple_of
+                if multiple_of and not isinstance(multiple_of, float):
```

Comment:
What's wrong with `multiple_of` being a float?

`y` is a multiple of `x` if `y=n*x`, where `n` is an int.
http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.2.1